client/src/pages/TablePage.jsx

```diff
@@ -47,10 +47,11 @@ const TablePage = () => {
   };
 
   const filterEntries = () => {
+    const noCap = (nm) => nm.replaceAll(' ', '').toLowerCase();
```

Comment:
This might cause issues because it will replace all spaces. I’d say trim is a better option since that only removes whitespace on the ends of the string